client/my-sites/post-selector/docs/example.jsx

```diff
@@ -17,10 +17,17 @@ const PostSelectorExample = React.createClass( {
 
 	getInitialState() {
 		return {
-			showTypeLabels: true
+			showTypeLabels: true,
+			selectedPostId: '',
```

Comment:
I think `null` (or `undefined`) would be a more sensible default for the selected post ID, since a true ID would be numeric. I changed this in 6fed690d7a38ffea1c4fa6bea0db2a02a81f2412. We could even leave this out of the initial state object and it would work just as well, albeit less clear.